src/pages/[quizId]/play.tsx

```diff
@@ -32,12 +39,23 @@ const Play = ({ channel }: { channel: any }) => {
   //   [channel, round.questions, stateIndex],
   // );
 
+  const handleSubmit = () => {
+    if (questionIndex < QUESTIONS.length - 1) {
```

Comment:
do not use mock data. later on it creates confusion

Reply:
Added comment instead